<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatRoomWindow</class>
 <widget class="QMainWindow" name="ChatRoomWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>601</width>
      <height>461</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Background: rgb(234, 230, 224);
border-radius: 5px;
border: 0px solid #535366;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>340</y>
       <width>541</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">Background: #FFFFFF;
border: 1.5px solid #535366;
border-radius: 0px; </string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="SendButton">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>20</y>
        <width>61</width>
        <height>32</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton { 
    background-color: #535366; 
    color: white; 
    border-radius: 5px; QPushButton { 
    background-color: #535366; 
    color: white; 
    border-radius: 5px; 
    }
 QPushButton:hover { 
    background-color: #555; 
    }
  QPushButton:pressed { 
    background-color: #777; 
    }
    }
 QPushButton:hover { 
    background-color: #555; 
    }
  QPushButton:pressed { 
    background-color: #777; 
    }</string>
      </property>
      <property name="text">
       <string>전송</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="MessageInput">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>441</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">Background: rgb(255, 255, 255);
border-radius: 5px;
border: 0px solid #535366;</string>
      </property>
     </widget>
    </widget>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>541</width>
       <height>311</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">Background: rgb(255, 255, 255);
border-radius: 5px;
border: 1.5px solid #535366;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BackButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>420</y>
       <width>61</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton { 
    background-color: #535366; 
    color: white; 
    border-radius: 5px; QPushButton { 
    background-color: #535366; 
    color: white; 
    border-radius: 5px; 
    }
 QPushButton:hover { 
    background-color: #555; 
    }
  QPushButton:pressed { 
    background-color: #777; 
    }
    }
 QPushButton:hover { 
    background-color: #555; 
    }
  QPushButton:pressed { 
    background-color: #777; 
    }</string>
     </property>
     <property name="text">
      <string>나가기</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
